src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -283,8 +283,8 @@ export class OperationDetails {
         return result.filter(x => !skipNames.includes(x));
     }
 
-    public async loadGatewayInfo(apiName: string): Promise<void> {
-        const hostnames = await this.apiService.getApiHostnames(apiName);
+    public async loadGatewayInfo(): Promise<void> {
+        const hostnames = await this.apiService.getHostnames();
```

Comment:
`this.apiService.getHostnames();` - some APIs maybe associated with different gateways, that's why we were passing the apiName before. Is it not the case anymore?

Reply:
no, it is not I guess. we have a simple endpoint /gateway/hostnames, and we are using the first of the array, it is not associated with any api.

Reply:
There is a resource `/apis/${apiName}/hostnames` - which returns hostnames associated with the specified API. Is was done like that because different APIs can have different gateways associated to it. By default, it's `Managed` gateway, but there might be more, each with it's own hostname:

![image](https://user-images.githubusercontent.com/2320302/178084002-498132e9-67f4-4088-8f08-b1759cb225df.png)

Reply:
hmm makes sense, in data api we dont have it tho, I will raise this in the channel